### ISSUE TITLE
Cuckoo: No screenshots available

### ISSUE THREAD

**Issue by Darjuw** [cuckoosandbox/cuckoo#1475]
For some unknown reason, I am not getting any screenshots from the latest Cuckoo package version.

I have verified that tesseract is indeed in /usr/bin for processing.conf
```
[screenshots]
enabled = yes
# Set to the actual tesseract path (i.e., /usr/bin/tesseract or similar)
# rather than "no" to enable OCR analysis of screenshots.
# Note: doing OCR on the screenshots is a rather slow process.
tesseract = /usr/bin/tesseract
```

I have verified that Pillow is installed on my guest using
```
import PIL
print PIL.PILLOW_VERSION
```

Any idea what I could be doing wrongly?

**Comment by SparkyNZL** [comment]
What cuckoo version are h running ?

**Comment by Darjuw** [comment]
Version 2.0.0, freshly downloaded from Github download.

**Comment by SparkyNZL** [comment]
Ok there is your problem, it's now a pip package

So if u use pip install cuckoo

This will install the right version and all the requirements , but all u
need is pil on ur vm and pillow on your host , tesseract-ocr is handy too
;)

**Comment by Darjuw** [comment]
Hmm I use the pip package too, where I can just launch Cuckoo with sudo cuckoo.
I have tried PIL on VM, and Pillow on host, but it's still not giving me any screenshots.

I am a bit confused here too, the [documentation](https://cuckoo.sh/docs/installation/guest_physical/requirements.html#install-python) is saying how I should install Pillow on the guest (Windows). 

None the less, Pillow/PIL isn't working for me for some reason hmm.

**Comment by SparkyNZL** [comment]
Oh don't run cuckoo as root

**Comment by Darjuw** [comment]
Hmm alright.

I ran the following, all without sudo this time. But Cuckoo is giving me permission denied errors everywhere :( 

Have I been doing something wrongly all these while? Everything works fine with sudo, just not the screenshot feature.

admin2@admin2-7010:~$ sudo cuckoo rooter -g admin2
admin2@admin2-7010:~$ cuckoo 

```
 Cuckoo Sandbox 2.0.1
 www.cuckoosandbox.org
 Copyright (c) 2010-2017

 Checking for updates...
 You're good to go!
No handlers could be found for logger "pip.utils"
Oops! Cuckoo failed in an unhandled exception!
Sometimes bugs are already fixed in the development release, it is therefore recommended to retry with the latest development release available https://github.com/cuckoosandbox/cuckoo
If the error persists please open a new issue at https://github.com/cuckoosandbox/cuckoo/issues

=== Exception details ===
Cuckoo version: 2.0.1
OS version: posix
OS release: Ubuntu 14.04 trusty
Python version: 2.7.6
Python implementation: CPython
Machine arch: x86_64
Modules: adium-theme-ubuntu:0.3.4 alembic:0.8.8 androguard:3.0 appdirs:1.4.3 apt-xapian-index:0.45 asn1crypto:0.22.0 beautifulsoup4:4.5.3 certifi:2017.1.23 cffi:1.10.0 chardet:2.3.0 click:6.6 colorama:0.3.7 command-not-found:0.3 cryptography:1.8.1 cuckoo:2.0.1 debtagshw:0.1 defer:1.0.6 dirspec:13.10 distorm3:3.3.4 distribute:0.7.3 django-extensions:1.6.7 django:1.8.4 dpkt:1.8.7 duplicity:0.6.23 ecdsa:0.13 elasticsearch:5.3.0 enum34:1.1.6 flask-sqlalchemy:2.1 flask:0.10.1 functools32:3.2.3.post2 html5lib:0.999 httplib2:0.8 httpreplay:0.2 idna:2.5 ipaddress:1.0.18 itsdangerous:0.24 jinja2:2.8 jsbeautifier:1.6.2 jsonschema:2.6.0 lockfile:0.8 lxml:3.6.0 m2crypto:0.24.0 mako:1.0.6 markupsafe:1.0 ndg-httpsclient:0.4.2 oauthlib:0.6.1 olefile:0.43 oletools:0.42 oneconf:0.3.7 packaging:16.8 pam:0.4.2 peepdf:0.3.4 pefile2:1.2.11 pexpect:3.1 pillow:3.2.0 pip:9.0.1 piston-mini-client:0.7.5 pyasn1:0.2.3 pycparser:2.17 pycrypto:2.6.1 pycups:1.9.66 pycurl:7.19.3 pydeep:0.2 pygobject:3.12.0 pymisp:2.4.54 pymongo:3.0.3 pyopenssl:17.0.0 pyparsing:2.2.0 pyserial:2.6 pysmbc:1.0.14.1 python-apt:0.9.3.5 python-dateutil:2.4.2 python-debian:0.1.21-nmu2ubuntu2 python-editor:1.0.3 python-magic:0.4.12 pythonaes:1.0 pyxdg:0.25 reportlab:3.0 requests:2.7.0 scapy:2.3.2 sessioninstaller:0.0.0 setuptools:35.0.1 sflock:0.2.7 six:1.10.0 software-center-aptd-plugins:0.0.0 sqlalchemy:1.0.8 ssh-import-id:3.21 system-service:0.1.6 tlslite-ng:0.6.0 twisted-core:13.2.0 twisted-web:13.2.0 typing:3.6.1 unity-lens-photos:1.0 urllib3:1.20 virtualenv:1.11.4 volatility:2.4 wakeonlan:0.2.2 werkzeug:0.12.1 wheel:0.29.0 xdiagnose:3.6.3build2 yara-python:3.5.0 zope.interface:4.0.5

Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/main.py", line 186, in main
    cuckoo_init(level, ctx)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/main.py", line 114, in cuckoo_init
    init_logging(level)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/startup.py", line 142, in init_logging
    init_logger("cuckoo.log", level)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/log.py", line 111, in init_logger
    l = logging.handlers.WatchedFileHandler(cwd("log", "cuckoo.log"))
  File "/usr/lib/python2.7/logging/handlers.py", line 391, in __init__
    logging.FileHandler.__init__(self, filename, mode, encoding, delay)
  File "/usr/lib/python2.7/logging/__init__.py", line 903, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python2.7/logging/__init__.py", line 928, in _open
    stream = open(self.baseFilename, self.mode)
IOError: [Errno 13] Permission denied: '/home/admin2/.cuckoo/log/cuckoo.log'
```

**Comment by SparkyNZL** [comment]
Because u have been running it as root you will need to change the
permission to ur cuckoo user , you can chown the .Cuckoo

**Comment by jbremer** [comment]
Has this issue been resolved?

**Comment by jbremer** [comment]
Will be closing shortly if no additional information is provided.

**Comment by Darjuw** [comment]
Hi, sorry for the late reply. Was overseas.

I did try doing a chown recursive to change owner from my current user (which I have been using for VMs and Cuckoo installation) to cuckoo, which I created as per the documentation requirement. However I am getting additional errors with Cuckoo rooter, web, and the package itself after the change.

I'll post the logs tomorrow when I get access to my PC.

**Comment by Darjuw** [comment]
Hi there,

I have tried to chown the .cuckoo on my admin2 user.
```
admin2@admin2-7010:~$ sudo chown -R cuckoo .cuckoo/
admin2@admin2-7010:~$ sudo chgrp -R cuckoo .cuckoo/
drwxrwsrwx 14 cuckoo cuckoo    4096 Apr 28 17:56 .cuckoo

```

And then I ran as per rooter requirements
`admin2@admin2-7010:~$ cuckoo rooter --sudo`

And then I ran Cuckoo without root
```
cuckoo@admin2-7010:/home/admin2$ cuckoo 

CuckooCriticalError: Error initializing machines: Vm file /home/admin2/Desktop/C/Win7/ActivatedWin7.vmx not found
```

For some reason it's giving me an error of VM not found, although the VM is clearly in that directory. It works previously when I ran as root with .cuckoo folder owner as admin2. Previously I have tried copying the VM folder to Cuckoo's home too and initializing from there, but it gave errors too. I can try again if needed.

**Comment by jbremer** [comment]
Are the `.vmx` and related files `admin2:admin2` too?

**Comment by Darjuw** [comment]
I have tried admin2:admin2 and cuckoo:cuckoo. Both gives the same error.

```
admin2@admin2-7010:~/Desktop$ sudo chown -R cuckoo C/Win7/
admin2@admin2-7010:~/Desktop$ sudo chgrp -R cuckoo C/Win7/

cuckoo@admin2-7010:/home/admin2$ cuckoo 

  .-----------------.
  | Cuckoo Sandbox? |
  |     OH NOES!    |\  '-.__.-'
  '-----------------' \  /oo |--.--,--,--.
                         \_.-'._i__i__i_.'
                               """""""""

 Cuckoo Sandbox 2.0.1
 www.cuckoosandbox.org
 Copyright (c) 2010-2017

 Checking for updates...
 You're good to go!
2017-05-04 16:26:30,680 [cuckoo.core.scheduler] INFO: Using "vmware" as machine manager
CuckooCriticalError: Error initializing machines: Vm file /home/admin2/Desktop/C/Win7/ActivatedWin7.vmx not found
```


I tried doing a chmod too.
```
cuckoo@admin2-7010:/home/admin2$ sudo chmod -R 777 Desktop/C/Win7
drwxrwxrwx  3 cuckoo cuckoo      28672 May  4 10:59 Win7
```

**Comment by jbremer** [comment]
What are the contents of the `/home/admin2/Desktop/C/Win7/` directory?

**Comment by Darjuw** [comment]
They're the VM files for the sandbox, made using VMWare. Here's a small extract of it.
```
admin2@admin2-7010:~$ ls -la /home/admin2/Desktop/C/Win7/ | head
total 35064684
drwxrwxrwx 3 cuckoo cuckoo      28672 May  5 13:39 .
drwxrwxrwx 6 admin2 admin2       4096 May  4 16:27 ..
-rwxrwxrwx 1 cuckoo cuckoo    1179648 Apr  4 16:23 ActivatedWin7-000001-s001.vmdk
-rwxrwxrwx 1 cuckoo cuckoo     720896 Apr  4 16:23 ActivatedWin7-000001-s002.vmdk
-rwxrwxrwx 1 cuckoo cuckoo     524288 Apr  4 16:22 ActivatedWin7-000001-s003.vmdk
-rwxrwxrwx 1 cuckoo cuckoo     983040 Apr  4 16:23 ActivatedWin7-000001-s004.vmdk
-rwxrwxrwx 1 cuckoo cuckoo     524288 Apr  4 16:22 ActivatedWin7-000001-s005.vmdk
-rwxrwxrwx 1 cuckoo cuckoo     524288 Apr  4 16:22 ActivatedWin7-000001-s006.vmdk
-rwxrwxrwx 1 cuckoo cuckoo     524288 Apr  4 16:22 ActivatedWin7-000001-s007.vmdk
```

I have solved the permissions issue now but doing a chmod on my admin2's desktop folder for Cuckoo.  But currently, while Cuckoo starts up without any error now, it does not seem to be accepting any submissions as I see no VMs starting.

I su'ed to Cuckoo and ran the following commands.

`cuckoo@admin2-7010:/home/admin2$ cuckoo rooter --sudo`

```
cuckoo@admin2-7010:/home/admin2$ cuckoo 

                                 _|
     _|_|_|  _|    _|    _|_|_|  _|  _|      _|_|      _|_|
   _|        _|    _|  _|        _|_|      _|    _|  _|    _|
   _|        _|    _|  _|        _|  _|    _|    _|  _|    _|
     _|_|_|    _|_|_|    _|_|_|  _|    _|    _|_|      _|_|

 Cuckoo Sandbox 2.0.1
 www.cuckoosandbox.org
 Copyright (c) 2010-2017

 Checking for updates...
 You're good to go!
2017-05-05 13:43:33,115 [cuckoo.core.scheduler] INFO: Using "vmware" as machine manager
```

```
cuckoo@admin2-7010:/home/admin2$ cuckoo submit /home/admin2/Downloads/npp.7.3.2.Installer.exe 
Success: File "/home/admin2/Downloads/npp.7.3.2.Installer.exe" added as task with ID #2
```

PS: Everything but screenshots works fine previously when I was using admin2's sudo to run Cuckoo. I installed Cuckoo using admin2, and the CWD is in /home/admin2.

**Comment by jbremer** [comment]
Why not show everything from that directory? E.g., I don't see no `ActivatedWin7.vmx` in it now ;-)
Looks like VMWare is hanging on listing the available VMs which means that something is going wrong with your VMs still.
My best advice would be to create some new VMs and don't touch `sudo` this time.

**Comment by jbremer** [comment]
Any update here @Darjuw?

**Comment by Darjuw** [comment]
Hi jbremer, as mentioned in my previous update, I have resolved the permissions issue now by doing a chmod on my admin2's desktop folder for Cuckoo. :D 

Here's the full log of it with the `ActivatedWin7.vmx`
`cuckoo@admin2-7010:/home/admin2$ ls /home/admin2/Desktop/C/Win7/
ActivatedWin7-000001-s001.vmdk  ActivatedWin7-000012.vmdk
ActivatedWin7-000001-s002.vmdk  ActivatedWin7-000013-s001.vmdk
ActivatedWin7-000001-s003.vmdk  ActivatedWin7-000013-s002.vmdk
ActivatedWin7-000001-s004.vmdk  ActivatedWin7-000013-s003.vmdk
ActivatedWin7-000001-s005.vmdk  ActivatedWin7-000013-s004.vmdk
ActivatedWin7-000001-s006.vmdk  ActivatedWin7-000013-s005.vmdk
ActivatedWin7-000001-s007.vmdk  ActivatedWin7-000013-s006.vmdk
ActivatedWin7-000001-s008.vmdk  ActivatedWin7-000013-s007.vmdk
ActivatedWin7-000001-s009.vmdk  ActivatedWin7-000013-s008.vmdk
ActivatedWin7-000001-s010.vmdk  ActivatedWin7-000013-s009.vmdk
ActivatedWin7-000001-s011.vmdk  ActivatedWin7-000013-s010.vmdk
ActivatedWin7-000001-s012.vmdk  ActivatedWin7-000013-s011.vmdk
ActivatedWin7-000001-s013.vmdk  ActivatedWin7-000013-s012.vmdk
ActivatedWin7-000001-s014.vmdk  ActivatedWin7-000013-s013.vmdk
ActivatedWin7-000001-s015.vmdk  ActivatedWin7-000013-s014.vmdk
ActivatedWin7-000001-s016.vmdk  ActivatedWin7-000013-s015.vmdk
ActivatedWin7-000001.vmdk       ActivatedWin7-000013-s016.vmdk
ActivatedWin7-000002-s001.vmdk  ActivatedWin7-000013.vmdk
ActivatedWin7-000002-s002.vmdk  ActivatedWin7-000014-s001.vmdk
ActivatedWin7-000002-s003.vmdk  ActivatedWin7-000014-s002.vmdk
ActivatedWin7-000002-s004.vmdk  ActivatedWin7-000014-s003.vmdk
ActivatedWin7-000002-s005.vmdk  ActivatedWin7-000014-s004.vmdk
ActivatedWin7-000002-s006.vmdk  ActivatedWin7-000014-s005.vmdk
ActivatedWin7-000002-s007.vmdk  ActivatedWin7-000014-s006.vmdk
ActivatedWin7-000002-s008.vmdk  ActivatedWin7-000014-s007.vmdk
ActivatedWin7-000002-s009.vmdk  ActivatedWin7-000014-s008.vmdk
ActivatedWin7-000002-s010.vmdk  ActivatedWin7-000014-s009.vmdk
ActivatedWin7-000002-s011.vmdk  ActivatedWin7-000014-s010.vmdk
ActivatedWin7-000002-s012.vmdk  ActivatedWin7-000014-s011.vmdk
ActivatedWin7-000002-s013.vmdk  ActivatedWin7-000014-s012.vmdk
ActivatedWin7-000002-s014.vmdk  ActivatedWin7-000014-s013.vmdk
ActivatedWin7-000002-s015.vmdk  ActivatedWin7-000014-s014.vmdk
ActivatedWin7-000002-s016.vmdk  ActivatedWin7-000014-s015.vmdk
ActivatedWin7-000002.vmdk       ActivatedWin7-000014-s016.vmdk
ActivatedWin7-000003-s001.vmdk  ActivatedWin7-000014.vmdk
ActivatedWin7-000003-s002.vmdk  ActivatedWin7-000015-s001.vmdk
ActivatedWin7-000003-s003.vmdk  ActivatedWin7-000015-s002.vmdk
ActivatedWin7-000003-s004.vmdk  ActivatedWin7-000015-s003.vmdk
ActivatedWin7-000003-s005.vmdk  ActivatedWin7-000015-s004.vmdk
ActivatedWin7-000003-s006.vmdk  ActivatedWin7-000015-s005.vmdk
ActivatedWin7-000003-s007.vmdk  ActivatedWin7-000015-s006.vmdk
ActivatedWin7-000003-s008.vmdk  ActivatedWin7-000015-s007.vmdk
ActivatedWin7-000003-s009.vmdk  ActivatedWin7-000015-s008.vmdk
ActivatedWin7-000003-s010.vmdk  ActivatedWin7-000015-s009.vmdk
ActivatedWin7-000003-s011.vmdk  ActivatedWin7-000015-s010.vmdk
ActivatedWin7-000003-s012.vmdk  ActivatedWin7-000015-s011.vmdk
ActivatedWin7-000003-s013.vmdk  ActivatedWin7-000015-s012.vmdk
ActivatedWin7-000003-s014.vmdk  ActivatedWin7-000015-s013.vmdk
ActivatedWin7-000003-s015.vmdk  ActivatedWin7-000015-s014.vmdk
ActivatedWin7-000003-s016.vmdk  ActivatedWin7-000015-s015.vmdk
ActivatedWin7-000003.vmdk       ActivatedWin7-000015-s016.vmdk
ActivatedWin7-000004-s001.vmdk  ActivatedWin7-000015.vmdk
ActivatedWin7-000004-s002.vmdk  ActivatedWin7-000016-s001.vmdk
ActivatedWin7-000004-s003.vmdk  ActivatedWin7-000016-s002.vmdk
ActivatedWin7-000004-s004.vmdk  ActivatedWin7-000016-s003.vmdk
ActivatedWin7-000004-s005.vmdk  ActivatedWin7-000016-s004.vmdk
ActivatedWin7-000004-s006.vmdk  ActivatedWin7-000016-s005.vmdk
ActivatedWin7-000004-s007.vmdk  ActivatedWin7-000016-s006.vmdk
ActivatedWin7-000004-s008.vmdk  ActivatedWin7-000016-s007.vmdk
ActivatedWin7-000004-s009.vmdk  ActivatedWin7-000016-s008.vmdk
ActivatedWin7-000004-s010.vmdk  ActivatedWin7-000016-s009.vmdk
ActivatedWin7-000004-s011.vmdk  ActivatedWin7-000016-s010.vmdk
ActivatedWin7-000004-s012.vmdk  ActivatedWin7-000016-s011.vmdk
ActivatedWin7-000004-s013.vmdk  ActivatedWin7-000016-s012.vmdk
ActivatedWin7-000004-s014.vmdk  ActivatedWin7-000016-s013.vmdk
ActivatedWin7-000004-s015.vmdk  ActivatedWin7-000016-s014.vmdk
ActivatedWin7-000004-s016.vmdk  ActivatedWin7-000016-s015.vmdk
ActivatedWin7-000004.vmdk       ActivatedWin7-000016-s016.vmdk
ActivatedWin7-000005-s001.vmdk  ActivatedWin7-000016.vmdk
ActivatedWin7-000005-s002.vmdk  ActivatedWin7-000017-s001.vmdk
ActivatedWin7-000005-s003.vmdk  ActivatedWin7-000017-s002.vmdk
ActivatedWin7-000005-s004.vmdk  ActivatedWin7-000017-s003.vmdk
ActivatedWin7-000005-s005.vmdk  ActivatedWin7-000017-s004.vmdk
ActivatedWin7-000005-s006.vmdk  ActivatedWin7-000017-s005.vmdk
ActivatedWin7-000005-s007.vmdk  ActivatedWin7-000017-s006.vmdk
ActivatedWin7-000005-s008.vmdk  ActivatedWin7-000017-s007.vmdk
ActivatedWin7-000005-s009.vmdk  ActivatedWin7-000017-s008.vmdk
ActivatedWin7-000005-s010.vmdk  ActivatedWin7-000017-s009.vmdk
ActivatedWin7-000005-s011.vmdk  ActivatedWin7-000017-s010.vmdk
ActivatedWin7-000005-s012.vmdk  ActivatedWin7-000017-s011.vmdk
ActivatedWin7-000005-s013.vmdk  ActivatedWin7-000017-s012.vmdk
ActivatedWin7-000005-s014.vmdk  ActivatedWin7-000017-s013.vmdk
ActivatedWin7-000005-s015.vmdk  ActivatedWin7-000017-s014.vmdk
ActivatedWin7-000005-s016.vmdk  ActivatedWin7-000017-s015.vmdk
ActivatedWin7-000005.vmdk       ActivatedWin7-000017-s016.vmdk
ActivatedWin7-000006-s001.vmdk  ActivatedWin7-000017.vmdk
ActivatedWin7-000006-s002.vmdk  ActivatedWin7-000018-s001.vmdk
ActivatedWin7-000006-s003.vmdk  ActivatedWin7-000018-s002.vmdk
ActivatedWin7-000006-s004.vmdk  ActivatedWin7-000018-s003.vmdk
ActivatedWin7-000006-s005.vmdk  ActivatedWin7-000018-s004.vmdk
ActivatedWin7-000006-s006.vmdk  ActivatedWin7-000018-s005.vmdk
ActivatedWin7-000006-s007.vmdk  ActivatedWin7-000018-s006.vmdk
ActivatedWin7-000006-s008.vmdk  ActivatedWin7-000018-s007.vmdk
ActivatedWin7-000006-s009.vmdk  ActivatedWin7-000018-s008.vmdk
ActivatedWin7-000006-s010.vmdk  ActivatedWin7-000018-s009.vmdk
ActivatedWin7-000006-s011.vmdk  ActivatedWin7-000018-s010.vmdk
ActivatedWin7-000006-s012.vmdk  ActivatedWin7-000018-s011.vmdk
ActivatedWin7-000006-s013.vmdk  ActivatedWin7-000018-s012.vmdk
ActivatedWin7-000006-s014.vmdk  ActivatedWin7-000018-s013.vmdk
ActivatedWin7-000006-s015.vmdk  ActivatedWin7-000018-s014.vmdk
ActivatedWin7-000006-s016.vmdk  ActivatedWin7-000018-s015.vmdk
ActivatedWin7-000006.vmdk       ActivatedWin7-000018-s016.vmdk
ActivatedWin7-000007-s001.vmdk  ActivatedWin7-000018.vmdk
ActivatedWin7-000007-s002.vmdk  ActivatedWin7-000019-s001.vmdk
ActivatedWin7-000007-s003.vmdk  ActivatedWin7-000019-s002.vmdk
ActivatedWin7-000007-s004.vmdk  ActivatedWin7-000019-s003.vmdk
ActivatedWin7-000007-s005.vmdk  ActivatedWin7-000019-s004.vmdk
ActivatedWin7-000007-s006.vmdk  ActivatedWin7-000019-s005.vmdk
ActivatedWin7-000007-s007.vmdk  ActivatedWin7-000019-s006.vmdk
ActivatedWin7-000007-s008.vmdk  ActivatedWin7-000019-s007.vmdk
ActivatedWin7-000007-s009.vmdk  ActivatedWin7-000019-s008.vmdk
ActivatedWin7-000007-s010.vmdk  ActivatedWin7-000019-s009.vmdk
ActivatedWin7-000007-s011.vmdk  ActivatedWin7-000019-s010.vmdk
ActivatedWin7-000007-s012.vmdk  ActivatedWin7-000019-s011.vmdk
ActivatedWin7-000007-s013.vmdk  ActivatedWin7-000019-s012.vmdk
ActivatedWin7-000007-s014.vmdk  ActivatedWin7-000019-s013.vmdk
ActivatedWin7-000007-s015.vmdk  ActivatedWin7-000019-s014.vmdk
ActivatedWin7-000007-s016.vmdk  ActivatedWin7-000019-s015.vmdk
ActivatedWin7-000007.vmdk       ActivatedWin7-000019-s016.vmdk
ActivatedWin7-000008-s001.vmdk  ActivatedWin7-000019.vmdk
ActivatedWin7-000008-s002.vmdk  ActivatedWin7-000020-s001.vmdk
ActivatedWin7-000008-s003.vmdk  ActivatedWin7-000020-s002.vmdk
ActivatedWin7-000008-s004.vmdk  ActivatedWin7-000020-s003.vmdk
ActivatedWin7-000008-s005.vmdk  ActivatedWin7-000020-s004.vmdk
ActivatedWin7-000008-s006.vmdk  ActivatedWin7-000020-s005.vmdk
ActivatedWin7-000008-s007.vmdk  ActivatedWin7-000020-s006.vmdk
ActivatedWin7-000008-s008.vmdk  ActivatedWin7-000020-s007.vmdk
ActivatedWin7-000008-s009.vmdk  ActivatedWin7-000020-s008.vmdk
ActivatedWin7-000008-s010.vmdk  ActivatedWin7-000020-s009.vmdk
ActivatedWin7-000008-s011.vmdk  ActivatedWin7-000020-s010.vmdk
ActivatedWin7-000008-s012.vmdk  ActivatedWin7-000020-s011.vmdk
ActivatedWin7-000008-s013.vmdk  ActivatedWin7-000020-s012.vmdk
ActivatedWin7-000008-s014.vmdk  ActivatedWin7-000020-s013.vmdk
ActivatedWin7-000008-s015.vmdk  ActivatedWin7-000020-s014.vmdk
ActivatedWin7-000008-s016.vmdk  ActivatedWin7-000020-s015.vmdk
ActivatedWin7-000008.vmdk       ActivatedWin7-000020-s016.vmdk
ActivatedWin7-000009-s001.vmdk  ActivatedWin7-000020.vmdk
ActivatedWin7-000009-s002.vmdk  ActivatedWin7.nvram
ActivatedWin7-000009-s003.vmdk  ActivatedWin7-s001.vmdk
ActivatedWin7-000009-s004.vmdk  ActivatedWin7-s002.vmdk
ActivatedWin7-000009-s005.vmdk  ActivatedWin7-s003.vmdk
ActivatedWin7-000009-s006.vmdk  ActivatedWin7-s004.vmdk
ActivatedWin7-000009-s007.vmdk  ActivatedWin7-s005.vmdk
ActivatedWin7-000009-s008.vmdk  ActivatedWin7-s006.vmdk
ActivatedWin7-000009-s009.vmdk  ActivatedWin7-s007.vmdk
ActivatedWin7-000009-s010.vmdk  ActivatedWin7-s008.vmdk
ActivatedWin7-000009-s011.vmdk  ActivatedWin7-s009.vmdk
ActivatedWin7-000009-s012.vmdk  ActivatedWin7-s010.vmdk
ActivatedWin7-000009-s013.vmdk  ActivatedWin7-s011.vmdk
ActivatedWin7-000009-s014.vmdk  ActivatedWin7-s012.vmdk
ActivatedWin7-000009-s015.vmdk  ActivatedWin7-s013.vmdk
ActivatedWin7-000009-s016.vmdk  ActivatedWin7-s014.vmdk
ActivatedWin7-000009.vmdk       ActivatedWin7-s015.vmdk
ActivatedWin7-000010-s001.vmdk  ActivatedWin7-s016.vmdk
ActivatedWin7-000010-s002.vmdk  ActivatedWin7-Snapshot18.vmem
ActivatedWin7-000010-s003.vmdk  ActivatedWin7-Snapshot18.vmsn
ActivatedWin7-000010-s004.vmdk  ActivatedWin7-Snapshot19.vmem
ActivatedWin7-000010-s005.vmdk  ActivatedWin7-Snapshot19.vmsn
ActivatedWin7-000010-s006.vmdk  ActivatedWin7-Snapshot20.vmem
ActivatedWin7-000010-s007.vmdk  ActivatedWin7-Snapshot20.vmsn
ActivatedWin7-000010-s008.vmdk  ActivatedWin7-Snapshot21.vmem
ActivatedWin7-000010-s009.vmdk  ActivatedWin7-Snapshot21.vmsn
ActivatedWin7-000010-s010.vmdk  ActivatedWin7-Snapshot23.vmem
ActivatedWin7-000010-s011.vmdk  ActivatedWin7-Snapshot23.vmsn
ActivatedWin7-000010-s012.vmdk  ActivatedWin7-Snapshot25.vmem
ActivatedWin7-000010-s013.vmdk  ActivatedWin7-Snapshot25.vmsn
ActivatedWin7-000010-s014.vmdk  ActivatedWin7-Snapshot27.vmem
ActivatedWin7-000010-s015.vmdk  ActivatedWin7-Snapshot27.vmsn
ActivatedWin7-000010-s016.vmdk  ActivatedWin7-Snapshot28.vmem
ActivatedWin7-000010.vmdk       ActivatedWin7-Snapshot28.vmsn
ActivatedWin7-000011-s001.vmdk  ActivatedWin7-Snapshot30.vmem
ActivatedWin7-000011-s002.vmdk  ActivatedWin7-Snapshot30.vmsn
ActivatedWin7-000011-s003.vmdk  ActivatedWin7-Snapshot31.vmem
ActivatedWin7-000011-s004.vmdk  ActivatedWin7-Snapshot31.vmsn
ActivatedWin7-000011-s005.vmdk  ActivatedWin7-Snapshot32.vmem
ActivatedWin7-000011-s006.vmdk  ActivatedWin7-Snapshot32.vmsn
ActivatedWin7-000011-s007.vmdk  ActivatedWin7-Snapshot33.vmem
ActivatedWin7-000011-s008.vmdk  ActivatedWin7-Snapshot33.vmsn
ActivatedWin7-000011-s009.vmdk  ActivatedWin7-Snapshot35.vmem
ActivatedWin7-000011-s010.vmdk  ActivatedWin7-Snapshot35.vmsn
ActivatedWin7-000011-s011.vmdk  ActivatedWin7-Snapshot36.vmem
ActivatedWin7-000011-s012.vmdk  ActivatedWin7-Snapshot36.vmsn
ActivatedWin7-000011-s013.vmdk  ActivatedWin7-Snapshot37.vmem
ActivatedWin7-000011-s014.vmdk  ActivatedWin7-Snapshot37.vmsn
ActivatedWin7-000011-s015.vmdk  ActivatedWin7-Snapshot38.vmem
ActivatedWin7-000011-s016.vmdk  ActivatedWin7-Snapshot38.vmsn
ActivatedWin7-000011.vmdk       ActivatedWin7-Snapshot39.vmem
ActivatedWin7-000012-s001.vmdk  ActivatedWin7-Snapshot39.vmsn
ActivatedWin7-000012-s002.vmdk  ActivatedWin7-Snapshot40.vmem
ActivatedWin7-000012-s003.vmdk  ActivatedWin7-Snapshot40.vmsn
ActivatedWin7-000012-s004.vmdk  ActivatedWin7-Snapshot8.vmem
ActivatedWin7-000012-s005.vmdk  ActivatedWin7-Snapshot8.vmsn
ActivatedWin7-000012-s006.vmdk  ActivatedWin7-Snapshot9.vmsn
ActivatedWin7-000012-s007.vmdk  ActivatedWin7.vmdk
ActivatedWin7-000012-s008.vmdk  ActivatedWin7.vmsd
ActivatedWin7-000012-s009.vmdk  ActivatedWin7.vmx
ActivatedWin7-000012-s010.vmdk  ActivatedWin7.vmx.backup
ActivatedWin7-000012-s011.vmdk  ActivatedWin7.vmxf
ActivatedWin7-000012-s012.vmdk  caches
ActivatedWin7-000012-s013.vmdk  vmware-0.log
ActivatedWin7-000012-s014.vmdk  vmware-1.log
ActivatedWin7-000012-s015.vmdk  vmware-2.log
ActivatedWin7-000012-s016.vmdk  vmware.log
`

Please refer to the previous post on the new problems I am encountering. Also, is it really necessary to do so much just to fix the no screenshot issue on the VMs? All other features worked fine.

**Comment by jbremer** [comment]
No, this issue was apparently you not being able to start the VM in the first place. I'm not really sure what the issue there is, though. How did you create this VM, did you copy it from somewhere else?

**Comment by Darjuw** [comment]
Perhaps I was not clear in stating my problem. I started this issue on Github because my Cuckoo report is not showing any screenshots after my analysis is done, despite having python, PIL and pillow installed on my host and guest. 

I only encountered the VM issue after I tried to follow the suggestion by @SparkyNZL to run things as Cuckoo, instead of admin2, which is the user I installed Cuckoo with. 

Everything on Cuckoo works fine but screenshots.
